posts/2024-05-02-comparison-of-performance.ipynb

```diff
@@ -0,0 +1,521 @@
+{
```

Comment:
<blockquote>almost inconce<strong>i</strong>vable that we wouldn't find what we need. </blockquote><p>Typo. Also where? Maybe a link to scipy docs as you've done for <code>numba</code> below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/coecms/coecms.github.io/pull/69/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>